gravitee-io/gravitee-access-management
# fix: allow to access id_token claims in UserMapper

fixes AM-1056

gravitee-io/issues#9381

## How to test

In FranceConnect IDP, add this additional parameter to get the AMR claims. This claims is present only into the id_token not in the UserInfo endpoint

![image](https://github.com/gravitee-io/gravitee-access-management/assets/3110552/e83b21bb-9f31-4836-a98b-8d21cf057301)

In the User Mapper linked to FranceConnect, add these entries. Note the amr_el one that will provide in the User profile the amr claims coming from the id_token.

![image](https://github.com/gravitee-io/gravitee-access-management/assets/3110552/fa3e1844-05ad-40ac-bbcb-783f1dc548d8)

in the user profile, you should see something like this

![image](https://github.com/gravitee-io/gravitee-access-management/assets/3110552/3e1b8e66-f626-4a67-8a0d-897153677c94)
